### DISCUSSION TITLE
Expanding model token limit

### DISCUSSION THREAD

**Discussion by Manicipal** [pashpashpash/vault-ai#48] [Ideas]
Is there a way to expand this model to larger token sizes? as the 32k token GPT4 API is being rolled out now. Could you let me know where to modify the code?

Thanks

**Comment by pashpashpash** [comment]
@Manicipal Good question! You would have to upload the model used to answer questions to gpt-4 in questions.go:
https://github.com/pashpashpash/vault-ai/blob/master/vault-web-server/postapi/questions.go#L81-L84

And then you would be able to increase the token limit in the buildPrompt function in openai.go:
https://github.com/pashpashpash/vault-ai/blob/master/vault-web-server/postapi/openai.go#L182
